src/AspNet.Security.OAuth.Odnoklassniki/OdnoklassnikiAuthenticationOptions.cs

```diff
@@ -0,0 +1,53 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Odnoklassniki
+{
+    /// <summary>
+    /// Configuration options for <see cref="OdnoklassnikiAuthenticationMiddleware"/>.
+    /// </summary>
+    public class OdnoklassnikiAuthenticationOptions : OAuthOptions
+    {
+        /// <summary>
+        /// Initializes a new <see cref="OdnoklassnikiAuthenticationOptions"/>.
+        /// </summary>
+        public OdnoklassnikiAuthenticationOptions()
+        {
+            AuthenticationScheme = OdnoklassnikiAuthenticationDefaults.AuthenticationScheme;
+            DisplayName = OdnoklassnikiAuthenticationDefaults.DisplayName;
+            ClaimsIssuer = OdnoklassnikiAuthenticationDefaults.Issuer;
+
+            CallbackPath = new PathString(OdnoklassnikiAuthenticationDefaults.CallbackPath);
+
+            AuthorizationEndpoint = OdnoklassnikiAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = OdnoklassnikiAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = OdnoklassnikiAuthenticationDefaults.UserInformationEndpoint;
+
+            Scope.Add("VALUABLE_ACCESS");
+            Scope.Add("GET_EMAIL");
+        }
+
+        public string ClientPublic { get; set; }
```

Comment:
Well, this one is another (besides `ClientId` and `ClientSecret`) parameter of an application. It is referred to as "application public key". I've named the property this way to be consistent with the other properties. I'm leaving it unchanged for now (only adding docs). Let me know if you require it to be named differently. May be `PublicKey` or `AppPublic`. Or even `AppPublicKey`.